platforms/hyperledger-besu/configuration/roles/create/certificates/ambassador/tasks/nested_main.yaml

```diff
@@ -102,14 +102,40 @@
     domain_name: "{{ node_name }}.{{ organizationItem.external_url_suffix }}"
   when: root_certs.failed == True
 
+# Generate crypto for besu nodes and knownserver file
+- name: create tls credentials for besu
+  shell: |
+    cd {{ ambassadortls }}
+    eval "openssl genrsa -out {{ node_name }}-besu-node.key 2048"
+    eval "openssl req -new -key {{ node_name }}-besu-node.key -out {{ node_name }}-besu-node.csr -subj '/CN={{ common_name }}'"
+    eval "openssl x509 -req -in {{ node_name }}-besu-node.csr -CA {{ rootca }}/rootca.pem -CAkey {{ rootca }}/rootca.key -CAcreateserial -out {{ node_name }}-besu-node.pem -days 500 -sha256"
+    eval "cat {{ node_name }}-besu-node.key {{ node_name }}-besu-node.pem > {{ node_name }}_certificates.pem"
+    eval "openssl pkcs12 -export -in {{ node_name }}_certificates.pem -out {{ node_name }}-besu-node.pkcs12 -password pass:somepass -name myAlias -noiter -nomaciter"
+    eval "echo "changeme" > {{ node_name }}-password"
+    eval "openssl x509 -in {{node_name}}-certchain.pem -noout -fingerprint -sha256 > {{node_name}}-certchain-sha256"
+    eval "export DIGEST=$(awk '{print $2}' {{node_name}}-certchain-sha256 | sed -n 's/Fingerprint=\(.*\)/\1/p')"
+    eval "echo "{{ node_name }}:{{ node.tm_clientport.port | default('8888') }} $DIGEST" > {{ node_name }}-knownServer"
+  vars:
+    common_name: "{{ node_name }}.{{ organizationItem.external_url_suffix }}"
```

Comment:
why is common_name removed from knownserver? I think both should be there

Reply:
As per the besu docs the trusted enclave file is in format "<hostname>:<port> <hexstring>", so I changed the hostname as to match the hostname in deployment. Should it be changed to common name and the same common name is used in deployment?